build.sh

```diff
@@ -16,7 +16,8 @@ mirror=${2:-"http://archive.ubuntu.com/ubuntu/"}
 # Install language with the most popcon
 gnomelanguage=${3:-'{en}'}	
 # Release name, used by debootstrap.  Examples: lucid, maverick, natty.
-release=${4:-zesty}
+#release=${4:-zesty}
```

Comment:
Can't we remove it?

Reply:
we can
but the build is still failing

Reply:
Keep in mind these are not only the files to change. You are getting your sources from the sources file.